hierarchicalforecast/utils.py

```diff
@@ -68,7 +68,75 @@ def cov2corr(cov, return_std=False):
     else:
         return corr
 
+# %% ../nbs/utils.ipynb 7
+# convert levels to output quantile names
+def level_to_outputs(level):
+    qs = sum([[50-l/2, 50+l/2] for l in level], [])
+    output_names = sum([[f'-lo-{l}', f'-hi-{l}'] for l in level], [])
+
+    sort_idx = np.argsort(qs)
+    quantiles = np.array(qs)[sort_idx]
+
+    # Add default median
+    quantiles = np.concatenate([np.array([50]), quantiles])
+    quantiles = torch.Tensor(quantiles) / 100
+    output_names = list(np.array(output_names)[sort_idx])
+    output_names.insert(0, '-median')
+    
+    return quantiles, output_names
+
+# convert quantiles to output quantile names
+def quantiles_to_outputs(quantiles):
+    output_names = []
+    for q in quantiles:
+        if q<.50:
+            output_names.append(f'-lo-{np.round(100-200*q,2)}')
+        elif q>.50:
+            output_names.append(f'-hi-{np.round(100-200*(1-q),2)}')
+        else:
+            output_names.append('-median')
+    return quantiles, output_names
+
 # %% ../nbs/utils.ipynb 8
+# given input array of sample forecasts and inptut quantiles/levels, 
+# output a Pandas Dataframe with columns of quantile predictions
+def _to_quantiles_df(samples, 
```

Comment:
Here is a draft for the docstring of this function, let's polish it:
""" Transform Samples into HierarchicalForecast input.

Auxiliary function to create compatible HierarchicalForecast input `Y_hat_df` dataframe.

**Parameters:**<br>
`samples`: numpy array. Samples from forecast distribution of shape [n_series, n_samples, horizon].<br>
`unique_ids`: .<br>
`dates`: .<br>
`quantiles`: .<br>
`level`: .<br>
`model_name`: .<br>

**Returns:**<br>
`Y_hat_df`: pd.DataFrame, With base quantile forecasts with columns `ds` and models to reconcile indexed by `unique_id`.<br>
"""